scss/jquery/overrides/_select2.scss

```diff
@@ -0,0 +1,246 @@
+
+      &.select2-search-choice-focus {
+        background: $gray-light;
+      }
+
+      /* stylelint-disable selector-max-compound-selectors, max-nesting-depth */
+      .select2-search-choice-close {
+        background: none !important;
+        left: auto;
+        right: 4px;
+        top: 4px;
+
+        &::before {
+          @include fa-icon();
+          background: none;
+          color: $brand-primary;
+          content: $fa-var-times-circle;
+          font-size: $font-size-base;
+          top: 1px;
+        }
+
+        &:hover {
+          &::before {
+            color: darken($brand-primary, $crm-darken-percentage);
+          }
+        }
+      }
+      /* stylelint-enable selector-max-compound-selectors, max-nesting-depth */
+    }
+
+    // The input field overlay
+    .select2-search-field {
+      /* stylelint-disable selector-max-compound-selectors */
+      input {
+        @include open-sans-font-family();
+        // These !important are necessary to counteract Bootstrap's base style
+        // when select2 is used inside the bootstrap namespace
+        font-size: $font-size-base !important;
+        line-height: normal !important;
+        margin: 0;
+        padding: 4px 4px 6px 8px;
+
+        // Spinner icon that appears when fetching data via ajax
+        &.select2-active { /* stylelint-disable-line max-nesting-depth, selector-no-qualifying-type */
+          background-position: center right 20px !important;
+        }
+      }
+      /* stylelint-enable selector-max-compound-selectors */
+    }
+  }
+}
+
+// The state of both single and multiple select2 when open
+.select2-dropdown-open {
+  .select2-choice,
+  .select2-choices, {
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0;
+    border-color: $input-border-focus;
+    box-shadow: none;
+  }
+
+  // Only for single select2, when open the arrow points up
+  .select2-choice {
+    .select2-arrow {
+      &::before {
+        content: $fa-var-caret-up;
+      }
+    }
+  }
+}
+
+// The select2 dropdown is appended right before </body>
+#{$outside-namespace-marker} {
+  // The dropdown list
+  .select2-drop.select2-drop-active.crm-container {
+    background: $crm-white;
+    border-bottom-left-radius: $border-radius-base;
+    border-bottom-right-radius: $border-radius-base;
+    border-color: $input-border-focus;
+
+    // CiviCRM-specific "refine search" controls
+    .crm-entityref-filters {
+      /* stylelint-disable selector-max-compound-selectors */
+      select,
+      input {
+        background: $crm-white;
+        border-color: $crm-grayblue-darker;
+        box-sizing: border-box;
+        height: $input-height-base;
+      }
+      /* stylelint-enable selector-max-compound-selectors */
+    }
+
+    // The list of items that match the search
+    .select2-results {
+      /* stylelint-disable selector-max-compound-selectors */
+      li {
+        color: $gray-dark;
+        padding: 5px;
```

Comment:
Same as above: 10px 20px etc...

Reply:
It's simply that the search items have some paddings instead of the default style which has none